event_routing_backends/handlers.py

```diff
@@ -0,0 +1,45 @@
+"""
+This module contains the handlers for signals emitted by the analytics app.
+"""
+import json
+import logging
+
+from django.dispatch import receiver
+from eventtracking.backends.event_bus import EventBusRoutingBackend
+from eventtracking.processors.exceptions import EventEmissionExit
+from eventtracking.tracker import get_tracker
+from openedx_events.analytics.signals import TRACKING_LOG_EVENT_EMITTED
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(TRACKING_LOG_EVENT_EMITTED)
+def send_tracking_log_to_backends(
+    sender, signal, **kwargs
+):  # pylint: disable=unused-argument
+    """
+    Listen for the TRACKING_LOG_EVENT_EMITTED signal and send the event to the enabled backends.
+    """
+    tracking_log = kwargs.get("tracking_log")
+
+    event = {
+        "name": tracking_log.name,
+        "timestamp": tracking_log.timestamp,
+        "data": json.loads(tracking_log.data),
+        "context": json.loads(tracking_log.context),
+    }
+
+    tracker = get_tracker()
+
+    engines = {
```

Comment:
Can you document what's happening here?

Reply:
Ready, can you take a look at the function docstring?